### ISSUE TITLE
about batch size

### ISSUE THREAD

**Issue by WuJunhui** [deepmind/kinetics-i3d#23]
Hi, 

In the paper _"Quo Vadis, Action Recognition? A New Model and the Kinetics Dataset"_, it is said you used 64 GPUs to train 3D ConvNets.
But in readme, you said minibatch size is 6. 
Does it mean that during multi-gpu training, the batch size is 6*64 = 384 ? 

Thanks！

**Comment by joaoluiscarreira** [comment]
Yes, that is correct, but note that batch norm layers use just the data in each gpu so it's not completely   equivalent to having a larger batch.

**Comment by liuqinying** [comment]
@joaoluiscarreira if minibatch size is 6 for each gpu, it seems to be too small for training batch norm layer because batch norm is useful only when batch size is big enough.So can you explain it in detail?

**Comment by joaoluiscarreira** [comment]
Note that all the layers are convolutional and batch norm accumulates statistics over the spatial and temporal dimensions so it's not as bad as if we would be talking about fully connected layers. I'm sure things would work out better if we could run batch norm over bigger batches but it's already ok.

**Comment by liuqinying** [comment]
@joaoluiscarreira thanks for your help.
